packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -298,12 +334,28 @@ export class Dialog
 
   @State() defaultMessages: DialogMessages;
 
+  @State() assistiveText = "";
+
+  @Watch("messages")
+  @Watch("dragEnabled")
+  @Watch("resizable")
+  updateAssistiveText(): void {
+    const { messages } = this;
+    this.assistiveText = messages
+      ? `${this.dragEnabled ? messages.dragEnabled : ""} ${this.resizable ? messages.resizeEnabled : ""}`.trim()
```

Comment:
Is `trim` necessary? I believe the text will be read correctly regardless. cc @geospatialem

Reply:
I can remove it. It shouldn't be necessary.